api/v1alpha3/virtualmachine_types.go

```diff
@@ -484,6 +488,17 @@ type VirtualMachineSpec struct {
 	// When the bootstrap provider is Cloud-Init, this value is used as the
 	// default value for spec.bootstrap.cloudInit.instanceID if it is omitted.
 	BiosUUID string `json:"biosUUID,omitempty"`
+
+	// +optional
+
+	// GuestID describes the desired short guest operating system identifier for
+	// a VM. If omitted, the VM may still be created with a guest ID if it is
+	// defined in the OVF template used to deploy this VM.
+	// Please refer to https://bit.ly/3yJdL0e for a complete list of supported
```

Comment:
Followed that link:
> Revision 14 March 2023  |  vSphere 8.0U1  |  vSphere vCenter SMS API 8.0U1

Maybe use [8.0U2](https://dp-downloads.broadcom.com/api-content/apis/API_VWSA_001/8.0U2/html/SDK/vsphere-ws/docs/ReferenceGuide/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html) instead? That doc also includes the since versions. Can also get the list supported by a live cluster using:
```console
% govc vm.option.info -cluster test-vpx-1717673857-26202-wcp.wcp-sanity-cluster
...
almalinux_64Guest           AlmaLinux (64-bit)
rockylinux_64Guest          Rocky Linux (64-bit)
windows2022srvNext_64Guest  Microsoft Windows Server 2025 (64-bit)
...
```
```

Reply:
Thanks! Updated with the newer version link.